include/d/a/obj/d_a_obj_island_LOD.h

```diff
@@ -8,7 +8,19 @@ class dAcOislandLOD_c : public dAcObjBase_c {
     dAcOislandLOD_c() {}
     virtual ~dAcOislandLOD_c() {}
 
+    virtual bool createHeap() override;
+    virtual int create() override;
+    virtual int doDelete() override;
+    virtual int actorExecute() override;
+    virtual int draw() override;
+
 private:
+    static const char* const resFilenames[5];
+    static const char* const resMdlNames[5];
+    
+    /* 0x330 */ nw4r::g3d::ResFile resFile;
+    /* 0x334 */ m3d::smdl_c mMdl;
+    /* 0x350 */ u8 lodModelIndex;
```

Comment:
```suggestion
    /* 0x330 */ nw4r::g3d::ResFile mResFile;
    /* 0x334 */ m3d::smdl_c mMdl;
    /* 0x350 */ u8 mMdlIdx;
```
Just to try and be consistent with the naming, m prefix for member variables and mixing/matching long short names. 

Note: The change I 'suggested' is just an example of what I mean. the main just looking for the m prefix